dnf5daemon-server/package.cpp

```diff
@@ -196,3 +196,158 @@ dnfdaemon::KeyValueMap package_to_map(
     }
     return dbus_package;
 }
+
+void add_string_list(json_object * json_pkg, const char * cattr, const std::vector<std::string> & vector) {
+    json_object * array = json_object_new_array();
+    json_object_object_add(json_pkg, cattr, array);
+    for (const auto & elem : vector) {
+        json_object_array_add(array, json_object_new_string(elem.c_str()));
+    }
+}
+
+std::string package_to_json(
+    const libdnf5::rpm::Package & libdnf_package, [[maybe_unused]] const std::vector<std::string> & attributes) {
```

Comment:
May I ask you why `[[maybe_unused]]` is used here?

Reply:
You may... forgotten from the early stages of development where the function was not fully implemented. Thanks for catching it!